styles.css

```diff
@@ -0,0 +1,1199 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@100;200;300;400;500;600;700;800;900&display=swap');
+
+:root {
+  --clr-purple: #6070ff;
+  --clr-white: #fff;
+  --clr-para: #344563;
+  --clr-gray-100: #b3bac5;
+}
+
+* {
+  margin: 0;
+}
+
+body {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  background-color: #fff;
+  margin: 0;
```

Comment:
- [ OPTIONAL ] Kindly make sure your design has the same background as the figma

 - Your background:
![backG](https://user-images.githubusercontent.com/97834160/206461660-f6813d8f-694f-485f-b8c7-d33ca492315d.png)

 - Figma's:
![BackGfig](https://user-images.githubusercontent.com/97834160/206461737-0516b18b-9acb-4cb5-bf9c-ed81418feec8.png)